src/main/java/frc/robot/commands/ClimberCommands.java

```diff
@@ -18,19 +18,18 @@
 
 /** A namespace for climber command factory methods. */
 public final class ClimberCommands {
-  private static final double CLIMB_ANGLE = Math.toRadians(-94);
-  private static final double STOW_ANGLE = Math.toRadians(90);
-  private static final double CLIMB_GROUND_INTAKE_ANGLE = Math.toRadians(97);
+  public static final double STOW_ANGLE = Math.toRadians(90);
+  public static final double GROUND_INTAKE_ANGLE_FOR_CLIMBING = Math.toRadians(97);
 
-  /** Returns a command that climbs. */
+  /** Returns a command that performs a Reefscape deep climb. */
   public static Command climb(Subsystems subsystems) {
     Climber climber = subsystems.climber;
     StatusLED statusLEDs = subsystems.statusLEDs;
     Arm coralArm = subsystems.coralArm;
 
     return Commands.parallel(
-            Commands.runOnce(() -> coralArm.setGoalAngle(STOWED.getArmAngle())),
-            Commands.runOnce(() -> climber.setGoalAngle(CLIMB_ANGLE)),
+            Commands.runOnce(() -> climber.setDeepClimbAngle(), climber),
+            Commands.runOnce(() -> coralArm.setGoalAngle(STOWED.getArmAngle()), coralArm),
```

Comment:
Should we use `CoralCommands.stowArm()` here? It does idle the `coralArm` subsystem and wait for the goal angle, but sinnce we're binding this command `whileTrue` that shouldn't matter.